### ISSUE TITLE
Accessible colors

### ISSUE THREAD

**Issue by feloy** [containers/podman-desktop#16138]
### Bug description

The pages using the xtermjs component (terminal, logs, etc) are displaying colors with not enough constrast, which fails WCAG contrast requirements.

https://xtermjs.org/docs/api/terminal/interfaces/iterminaloptions/#optional-minimumcontrastratio is an option which could be used to automatically increase the contrast at a level compatible with WCAG requirements.

### Operating system

all

### Installation Method

None

### Version

1.25.0

### Relevant log output

```shell

```
